### DISCUSSION TITLE
Error when trying to make an update to the resource   {"error":{"clientVersion":"2.23.0"}}

### DISCUSSION THREAD

**Discussion by arunkumar413** [prisma/prisma#7555] [Q&A]
I'm getting the below error when I try to make an update to a resource.  Unfortunately there isn't any debug information. The error message is passed as a response object
`{"error":{"clientVersion":"2.23.0"}} `

```ts
exports.moveReferralToProspect = async function (req, res) {
  try {
    const result = await prisma.user.update({ where: { id: req.params.id }, data: { status: "verified" } });
    return res.status(200).json(result);
  } catch (err) {
    res.json({ error: err });
  }
};

```

**Comment by ryands17** [comment]
@arunkumar413 👋 

It seems that you have found the solution on Slack.

In such cases, I would suggest printing the error and not just relying on the response sent to the client. For e.g.

```ts
exports.patchitem= async function (req, res) {
  try {
    const result = await prisma.user.update({ where: { id: req.params.id }, data: { status: "verified" } });
    console.log("############################ Result ############################");
    console.log(result);
    return res.status(200).json(result);
  } catch (err) {
    console.log('patchitem', err);
    res.json({ error: err });
  }
};
```

This will make sure the error is properly printed.

**Reply by arunkumar413** [reply]
@ryands17  There was no console error message in this case. The error that was caught by the catch block was sent as a response.

**Reply by ryands17** [reply]
Yes but the `console.log` in `catch` will print something in the backend so that should be available.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
